composites/Plugin/Shared/components/Button.js

```diff
@@ -268,15 +268,23 @@ IconButton.defaultProps = {
  * @returns {ReactElement} Styled icon button.
  */
 export const IconsButton = ( props ) => {
-	const { children: text, prefixIcon, prefixIconColor, suffixIcon, suffixIconColor } = props;
-
-	const newProps = omit( props, [ "prefixIcon", "prefixIconColor", "suffixIcon", "suffixIconColor" ] );
+	const {
+		children: text,
+		prefixIcon,
+		prefixIconColor,
+		suffixIcon,
+		suffixIconColor,
+		prefixIconViewBox,
```

Comment:
This shouldn't exist, can't we update the SVG icon to have a `1720x1720` viewbox as well? All these `iconViewbox` props are really cluttering the code. If it's not possible to change the viewbox of the icon, we should update the viewBox of the icons that require a different view box in the `SvgIcon` component.